libbackscrub.h

```diff
@@ -0,0 +1,54 @@
+/* This is licenced software, @see LICENSE file.
+ * Authors - @see AUTHORS file.
+==============================================================================*/
+
+// tested against tensorflow lite v2.4.1 (static library)
+#ifndef _LIBBACKSCRUB_H
+#define _LIBBACKSCRUB_H
+
+// for cv::Mat and related types
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <stdarg.h>
+
+#define TFLITE_MINIMAL_CHECK(x)                              \
```

Comment:
There should be no need to export this symbol to users of our library. If there is, the abstraction is broken.

Reply:
Agreed - another hangover from original code, please bring your own assertion checker outside the library :smile: